### ISSUE TITLE
rss issue + request to be added

### ISSUE THREAD

**Issue by tlienart** [kescobo/blog.bonham.ch#3]
Hello Kevin:

1. in the build action you can see that there's a ton of warnings like

```
┌ Warning: {{ ... }}
│ ---------
│ A block '{{website_title}}' was found but the name 'website_title' does not
│ correspond to a built-in block or hfun nor does it match anything
│ defined in `utils.jl`. It might have been misspelled.
└ @ Xranklin ~/.julia/packages/Xranklin/ELeJI/src/convert/postprocess/dbb.jl:168
```

This is a pretty simple fix, just add `website_title` in your config.md. More broadly though, I think the RSS build of your website is broken, I didn't look into it but maybe either put `generate_rss = false` or we should try to figure out what are the issues (good test case again!)

2. since you seem to like testing the new stuff with F/Xranklin (for which I'm very grateful), you can now specify all that is in `config.md` in a `config.jl` file (and get proper syntax highlighting as a result), just remove the `+++` and for the `\newcommand` definition at the bottom, if you're still using it, you can declare it with

```
# (...)
fn_title = ""
ignore=["_drafts/"]
lx"""
\newcommand{\html}[1]{~~~#1~~~}
"""
# end of config.jl
```

3. would you mind adding me to this repo? I won't do anything else than triggering the build action from time to time and experimenting with using the julia cache action for instance. With close to 400 contexts, your website is a great test case and we should really be able to bring down the build time under 10 minutes. I could just fork your repo but thought that at the same time I might try to do very light fixes here and there if I see stuff that can be fixed and that it'd be easier that way. Thanks!

**Comment by kescobo** [comment]
> More broadly though, I think the RSS build of your website is broken

Yeah, it's on my list to look at :laughing: . Low priority at the moment, but it would be great

>since you seem to like testing the new stuff with F/Xranklin (for which I'm very grateful), you can now specify all that is in config.md in a config.jl file (and get proper syntax highlighting as a result),

Oh cool! Will do. Commands are from the template, not using them ATM

>would you mind adding me to this repo? 

I think you are? Do you need increased privileges?

**Comment by kescobo** [comment]
Fixed some of this here: [34e281a](https://github.com/kescobo/blog.bonham.ch/commit/34e281a8e1e14b6f8fd6ba0ad37279d3a09a3fb5)
